recipes/bamhash/1.1/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export LDFLAGS=-L$PREFIX/lib
+export CXXFLAGS=-I$PREFIX/include
+
+make all
+
+mv bamhash_checksum_bam $PREFIX/bin
+mv bamhash_checksum_fastq $PREFIX/bin
```

Comment:
Yeah, I just realized it is because the 1.0 version doesn't `mv bamhash_checksum_fasta`.
 
Apparently whatever is happening is only locally (sorry for all that).

I'm getting this from `sudo conda build recipes/bamhash/1.1`:
```
+ bamhash_checksum_fasta --version
/home/hassan/anaconda3/conda-bld/1.1_1473152829350/test_tmp/run_test.sh: line 5: bamhash_checksum_fasta: command not found
```
Even though I see the following somewhere in conda-bld:
```
[hassan@hassan-pc bin]$ ls
activate                gappletviewer  gjarsigner     openssl
aot-compile             gc-analyze     gjavah         rebuild-gcj-db
bamhash_checksum_bam    gcc            gjdoc          x86_64-unknown-linux-gnu-c++
bamhash_checksum_fasta  gcc-ar         gkeytool       x86_64-unknown-linux-gnu-g++
bamhash_checksum_fastq  gcc-nm         gnative2ascii  x86_64-unknown-linux-gnu-gcc
c++                     gcc-ranlib     gorbd          x86_64-unknown-linux-gnu-gcc-4.8.5
cc                      gcj            grmic          x86_64-unknown-linux-gnu-gcc-ar
cloog                   gcj-dbtool     grmid          x86_64-unknown-linux-gnu-gcc-nm
conda                   gcjh           grmiregistry   x86_64-unknown-linux-gnu-gcc-ranlib
cpp                     gcov           gserialver     x86_64-unknown-linux-gnu-gcj
c_rehash                gfortran       gtnameserv     x86_64-unknown-linux-gnu-gfortran
deactivate              gij            jcf-dump
g++                     gjar           jv-convert
[hassan@hassan-pc bin]$ ./bamhash_checksum_fasta --version
bamhash_checksum_fasta version 1.1
```

My guess is that bamhash_checksum_fasta is not being copied to _t_env directory with the other 2 files, where conda performs its tests.
```
[hassan@hassan-pc 1.1_1473153246074]$ cd _t_env/bin/
[hassan@hassan-pc bin]$ ls
activate              bamhash_checksum_fastq  c_rehash    openssl
bamhash_checksum_bam  conda                   deactivate
[hassan@hassan-pc bin]$ 
```